### DISCUSSION TITLE
Encouragement to be part of Social coding commons

### DISCUSSION THREAD

**Discussion by aschrijver** [magicstone-dev/magicstone.dev#45] [Process]
This relates to [earlier feedback](https://github.com/magicstone-dev/magicstone.dev/issues/11) I provided at the time of inception of Magic Stone, and the community currently pondering new directions (whether or not to reboot based on a greenfield project).

---

Since I [mutually introduced](https://matrix.to/#/!lMZKxUdxFWJeNKQwxV:matrix.org/$ZwgLjwXItsC8tYuXR-_XjY7y4t9z6OiwNRx1Z2vC2Jg?via=matrix.org&via=halogen.city&via=darknight-coffee.org) you to each other, I really _love_ ❤️ to see the exchange of thoughts taking place on chat between Magic Stone and Guild Alpha.

My method of advocacy and community work involves a lot of what I call ["weaving in public"](https://discuss.coding.social/t/weaving-in-public-connecting-people-and-interests/85). One frustration - which I turned into my passion - is to notice the individualism and self-centeredness that often see people reinvent wheels rather than collaborate.

My insight was that this is only natural, and that any larger collaborative network in a grassroots movement  should take these social dynamics into account. That is my focus now, and where I spend much energy. To facilitate any initiative's full independence yet value-exchange with other initiative in ways that lead to synergy. Key to that is realising that it is intrinsic motivation and other hedonistic aspects that drive people to do what they do. Simply put: Everyone does what they like the most.

My passion is considering high-level, strategic and holistically how grassroots systems may overcome weaknesses and inherent unsustainability (for their participants). Yours - Guild Alpha and Magic Stone - is of a practical and pragmatic focus, directly related to the "Joy of Coding" (note: a hedonistic motive). When you talk about what makes these dev processes tick, that fits neatly in that higher level framework.

What I would like to propose to David Sterry and the folks in this initiative, is to consider yourself part of Social Coding Movement. Join the DoOcracy. SC movement is explicitly NOT a community, but a loose conglomerate of otherwise independent initiatives. The common theme of the movement is anything related to the Free Software Development Lifecycle (FSDL), scoped to ANY activity from birth till death of a FOSS project. With a focus on (holistic) sustainability.

Being part of the movement thus merely means find where Magic Stone likes to fit in, and shape a collaboration with Guild Alpha in any way you want. And with others joining the movement. For instance another delightful community I intend to approach with a similar proposal: [Malleable Systems](https://malleable.systems/) community. There's already alignment in the discusson on "ease of forking" above.

The way I view Magic Stone is it consisting of two parts: 1) C4 The Process (theory) and 2) C4 The Projects (practice). Point 1) is where Magic Stone and SC movement's theme coincide. The technical discussion of a "feature XYZ" is out of scope for the movement, but a process improvement isn't. In our collab SC movement can offer her tools and channels where the scope matches. Most notable by offering [forum space](https://discuss.coding.social/) for Magic Stone in a dedicated category for Magic Stone: The Process. You have a discussion space on Github which you can then reserve for the more technical stuff within Magic Stone.

**Comment by weex** [comment]
Thanks @aschrijver for all your work and helping to get people connected through common and self-interest alike.

As you say Social Coding is explicitly not a community and I like the de-emphasis of any kind of formal membership. We can work together toward similar goals without pledges or ceremony. I agree also with the concept of a DoOcracy, leading with action and attempting to replace politics with protocol whenever possible.

I think we're at a good point to collaborate in the future, especially as any greenfield Magic Stone projects sprout. I'm personally in exploration and hacking mode and wrote up my current idea in [The Global Town Square is for Everyone](https://davidsterry.com/blog/2023/09/the-global-town-square-is-for-everyone-not-just-blue-checks/). I encourage all to explore and discuss what excites them in FOSS and social media as well.

**Comment by aschrijver** [comment]
Hi @weex :wave: 

In follow-up to this proposal, which is timeless and still open, I think you might be interested to read up on the current status of what is currently called [Social coding commons](https://coding.social). The website provides an introduction and overview of [Social experience design](https://coding.social/introduction) to the public, while activities in the movement are done by 'working in commons' between proactive participants.

<img width="1491" height="699" alt="Text 'Welcome to Social coding commons' followed by a diagram that depicts the scope of SX as three expanding elipses. Innermost is Developer experience (DX) which is suitable to work with and create technical systems. Surrounding it is User experience (UX), which includes DX in our scope definition, and allows us to deliver socio-technical platforms. Outermost elipsis is Social experience design (SX) that holistically emcompasses both DX and SX and allows us to create socio-cultural solutions. SX takes us from individual needs, via inter-personal relationships, to leaving societal imprint." src="https://github.com/user-attachments/assets/e7db458c-f87c-408e-bae1-4883c89a48d4" />
